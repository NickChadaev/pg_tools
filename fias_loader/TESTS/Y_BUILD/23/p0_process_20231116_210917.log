2023-11-16 21:09:17 home/n.chadaev@abrr.local
127.0.0.1 5432 unsi_test_23 postgres /home/n.chadaev@abrr.local/Y_BUILD/23/stage_gar_c_23.csv 2023-11-13
--------------------------------------------------------
[21:09:17] ... Краснодарский край, id_region: 24, Execute: /home/n.chadaev@abrr.local/Y_BUILD/23/stage_gar_c_23.csv, process_id:25345
[21:09:17] ... Start process
[21:09:17] ...  -- Version
[21:09:17] ...  -- Очистка от данных
[21:09:18] ...  -- Убираем индексы
[21:09:18] ...  -- Set UNLOGGED
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_APARTMENT_TYPES_20231113_78e8cff6-619c-4b2e-806f-6c3c18c41d8c.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDR_OBJ_TYPES_20231113_15f2ec37-c925-4252-b49a-5473d975a16f.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ROOM_TYPES_20231113_32952955-cd74-40c1-bc84-06d94fe9559d.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OPERATION_TYPES_20231113_5151274c-91f4-4286-aaff-fc37112b660a.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_PARAM_TYPES_20231113_fe767969-20f3-4044-9ef6-67b4cb2881ce.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_HOUSE_TYPES_20231113_507d1000-4460-4d2f-a5b1-2ab5b9cd8a26.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_ADDHOUSE_TYPES_20231113_d99b31fe-7b05-400a-ac92-23919e58da07.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_OBJECT_LEVELS_20231113_b5fa7b3d-ea26-40c9-8b3a-9b986261ea66.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_TYPES_20231113_b8068ed2-dd18-4365-abb7-3dbd98698c35.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/AS_NORMATIVE_DOCS_KINDS_20231113_2f0f38c6-d598-4fd7-b325-2aa6aca29ed2.XML
[21:09:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_REESTR_OBJECTS_20231113_13db02b5-cda4-4090-8901-e064e191c0b1.XML
[22:06:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_ADDR_OBJ_20231113_3f813524-068d-42a8-a0ce-4a5f9a94e411.XML
[22:07:18] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_HOUSES_20231113_9b9955f7-51c0-4d0d-ad3b-5a700b0eef02.XML
[22:59:04] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_STEADS_20231113_5c0b73c3-5cfd-4e3c-92f7-3187def9c44e.XML
[23:10:14] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_APARTMENTS_20231113_d2d0e306-d97c-4922-a55e-b2cde96ae3f9.XML
[23:32:29] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_ROOMS_20231113_f15826fe-d4b9-4da6-9d64-8b03f934f0c4.XML
[23:32:33] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_CARPLACES_20231113_e5028425-79c5-410a-9a5c-8c7bfe3ebe7d.XML
[23:33:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_ADM_HIERARCHY_20231113_9025151c-7936-4a2d-a269-3f358cdef04b.XML
[00:35:11] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_MUN_HIERARCHY_20231113_a5aa876e-eb56-47a1-a5da-82448f8c9dad.XML
[01:33:30] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_CHANGE_HISTORY_20231113_8e5a65d2-a62d-45a5-bc4f-ffb465e38dad.XML
[04:16:09] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_ADDR_OBJ_DIVISION_20231113_91fba985-1b3b-4f70-8499-0b5032cf6e8e.XML
[04:16:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_ADDR_OBJ_PARAMS_20231113_6e35e6f4-3284-4c46-b894-cf957404ae0c.XML
[04:25:22] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_STEADS_PARAMS_20231113_133aad89-e7cb-45ea-a1f2-8bdc65cafb0d.XML
[06:15:48] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_HOUSES_PARAMS_20231113_e76cad5e-cbbb-479b-9324-9a82789518a9.XML
[11:02:35] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_APARTMENTS_PARAMS_20231113_6ad2836c-cc25-476b-a143-6351172c68d6.XML
[11:40:10] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_ROOMS_PARAMS_20231113_b1b5691e-bd5f-4526-93b0-41f867684bf0.XML
[11:40:14] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_CARPLACES_PARAMS_20231113_390acead-30ee-48ec-b93b-520aaf3fef70.XML
[11:40:16] ... /home/n.chadaev@abrr.local/7_DATA/FIAS_GAR_2023_11_13/23/AS_NORMATIVE_DOCS_20231113_10fc192d-aa8e-48fc-874c-66a7842cdc25.XML
[11:52:16] ...  -- Data finish
[11:52:16] ...  -- Set LOGGED
[11:54:21] ... Stop process
--------------------------------------------------------
