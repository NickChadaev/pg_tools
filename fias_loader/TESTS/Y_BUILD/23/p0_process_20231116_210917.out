 save_gar_version 
------------------
               82
(1 row)

CALL
CALL
CALL
 set_gar_dt_create 
-------------------
                82
(1 row)

CALL
gar_fias.as_apartment_type: 13
gar_fias.as_addr_obj_type: 421
gar_fias.as_room_type: 3
gar_fias.as_operation_type: 27
gar_fias.as_param_type: 20
gar_fias.as_house_type: 14
gar_fias.as_add_house_type: 4
gar_fias.as_object_level: 17
gar_fias.as_norm_docs_types: 25
gar_fias.as_norm_docs_kinds: 4
gar_fias.as_reestr_objects: 4552991
gar_fias.as_addr_obj: 70925
gar_fias.as_houses: 3861881
gar_fias.as_steads: 872472
gar_fias.as_apartments: 1757712
gar_fias.as_rooms: 4872
gar_fias.as_carplaces: 48208
gar_fias.as_adm_hierarchy: 4859033
gar_fias.as_mun_hierarchy: 4812345
gar_fias.as_change_history: 13521196
gar_fias.as_addr_obj_division: 1563
gar_fias.as_addr_obj_params: 754830
gar_fias.as_steads_params: 8896143
gar_fias.as_houses_params: 22762539
gar_fias.as_apartments_params: 3076622
gar_fias.as_rooms_params: 5604
gar_fias.as_carplaces_params: 2548
gar_fias.as_normative_docs: 930903
